docker/xgo-image-deb6/beats-builder/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM tudorg/xgo-deb6-1.4.2
```

Comment:
@electrical How did you do it in the past, did you use your personal account for hub.docker.com images?